_posts/2022-04-25-tardigrade-launch.md

```diff
@@ -0,0 +1,301 @@
+below issues:
+
+* Reliable landing times: Queries that run for hours can fail and restart 
+  from scratch. This wastes CPU resources and makes it hard for you to meet 
+  your data landing SLA. 
+* Cost-efficient clusters: Trino queries that need terabytes of distributed 
+  memory require extremely large clusters today due to lack of iterative 
+  execution.
+* Concurrency: Multiple independent clients may submit their queries 
+  concurrently. Due to the lack of available resources at a certain moment some
+  of these queries may need to be killed and restarted from zero after a 
+  while. This makes the landing time hardly predictable.
+
+While it is certainly possible to 
+[structure your workload to avoid these problems](https://engineering.salesforce.com/how-to-etl-at-petabyte-scale-with-trino-5fe8ac134e36) 
+it might not be easy to maintain for every company and may not always be
+possible for certain use cases.
+
+Project Tardigrade is designed to provide an "out of the box" solution for the
+aforementioned problems. We’ve created an innovative architecture (you can read
+about it [here](https://docs.google.com/document/d/18QQy8dkZQDNiBPEe3bsyt3RhImCKye4fUJB8bvoeLBI)
+and track ongoing developments 
+[here](https://github.com/trinodb/trino/issues/9101)) that allows us to 
+implement an advanced resource aware scheduling with granular retries. 
+What this means is:
+* When your long-running queries experience a failure, they don’t have to start
+  from scratch
+* When queries require more memory than currently available in the cluster they
+  would still be able to succeed
+* When multiple queries are submitted concurrently they would be able to share
+  resources in a fair way and make steady progress
+
+Trino will do all the hard work behind the scenes of allocating, configuring,
+and maintaining the queries. Instead of spending time tuning Trino clusters to
+match your workload requirements or reorganizing your workload to match your
+Trino cluster capabilities you spend your time doing analytics and delivering
+business value. And most importantly, your heart won’t skip a beat when you
+wake up in the morning wondering whether that query landed on time.
+
+## What Have We Done To Test It Out?
+
+Since there's no publicly available testing query set for ETL use cases, we
+handcrafted hundreds of ETL-like queries based on the TPC-H and TPC-DS datasets.
+
+To simulate real world settings, we deployed a Tardigrade cluster of 15
+`m5.8xlarge` nodes and repeatedly executed thousands of queries over datasets
+of different sizes (`10GB` / `1TB` / `10TB`). The queries were executed
+sequentially as well as with concurrency factors of 5, 10, and 20. Failure
+recovery capabilities were tested by crashing a random node in a cluster every
+couple of minutes while streaming a live workload. 
+
+## How Do I Enable Granular Retries?
+
+Tardigrade can only be enabled for an entire cluster.
```

Comment:
"Fault tolerant execution can only be enabled ...". See comment about regarding using Tardigrade as the name of a feature.